cssom/Overview.bs

```diff
@@ -1457,10 +1462,35 @@ To <dfn>serialize a CSS rule</dfn>, perform one of the following in accordance w
 
 
  <dt>{{CSSMediaRule}}
- <dd>Issue: ...
+ <dd>
+  The result of concatenating the following:
+  <ol>
+   <li>The string "<code>@media</code>" followed by a single SPACE (U+0020).</li>
+   <li>The result of performing <a>serialize a media query list<a> on rule's media query list.</li>
+   <li>A single SPACE (U+0020), followed by the string "{", i.e., LEFT CURLY BRACKET (U+007B), followed by a single SPACE (U+0020).</li>
```

Comment:
It seems browsers use a newline and then indent the child rules by two spaces...

http://software.hixie.ch/utilities/js/live-dom-viewer/saved/4682